integrations/access/servicenow/app.go

```diff
@@ -403,15 +403,19 @@ func (a *App) tryApproveRequest(ctx context.Context, req types.AccessRequest) er
 		logger.Get(ctx).Debugf("Skipping the approval: %s", err)
 		return nil
 	}
+	log.Debugf("Checking the following shifts to see if the requester is on-call: %s", serviceNames)
```

Comment:
Can we use `log.WithFields` to gradually enriching the log context?
For instance, I think we should always print the `req.GetName()`.

Reply:
If that's OK, I will send a PR later that harmonizes all logs in all plugins to be more verbose and set fields.